labgrid/remote/client.py

```diff
@@ -915,6 +915,29 @@ def flashscript(self):
 
         drv = self._get_driver_or_new(target, "FlashScriptDriver", name=name)
         drv.flash(script=self.args.script, args=self.args.script_args)
+    
+    def debugger(self):
+        place = self.get_acquired_place()
+        target = self._get_target(place)
+        name = self.args.name
+        from ..resource.lauterbach import (NetworkLauterbachDebugger, NetworkUSBLauterbachDebugger)
+        from ..resource.udev import (USBLauterbachDebugger)
+
+        drv = None
+        try:
+            drv = target.get_driver("DebuggerProtocol", name=name)
```

Comment:
Ah, that's where the protocol is useful. I guess until a second debugger driver is added, we can simply reference the driver directly.